recipes/args/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: args
+  version: {{ version }}
+
+source:
+  fn: args-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/e5/1c/b701b3f4bd8d3667df8342f311b3efaeab86078a840fb826bd204118cc6b/args-{{ version }}.tar.gz
+  md5: 66faf79ba2511def7b8b81d542482046
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - args
+
+about:
+  home: https://github.com/kennethreitz/args
+  license: BSD License
```

Comment:
I do hope that this is that: https://github.com/kennethreitz/args/blob/master/LICENSE